test/com/xilinx/rapidwright/design/TestDesign.java

```diff
@@ -142,4 +149,57 @@ public void testRouteThruPIPsSitePinInsts() {
             Assertions.assertTrue(spi.getSite().isInputPin(spi.getName()));
         }
     }
+
+    private EDIFNetlist generateEDIF(String edifName, long numLibraries, long cellsPerLibrary, long netsPerCell) {
+        EDIFNetlist netlist = new EDIFNetlist(edifName);
+        EDIFDesign edifDesign = new EDIFDesign(edifName);
+        edifDesign.addProperty(EDIFTools.EDIF_PART_PROP.toLowerCase(), Device.AWS_F1);
+        netlist.setDesign(edifDesign);
```

Comment:
Some of this functionality might be covered by `EDIFTools.createNewNetlist(String topName)` and `EDIFTools.ensureCorrectPartInEDIF(EDIFNetlist edif, String partName)`